### ISSUE TITLE
Didn't get the same results as you

### ISSUE THREAD

**Issue by Haiduongcable** [lorenmt/reco#17]
I retrain your model in Pascal Voc in full label setting with semi supervised + reco loss + class mix with 600 labels, but I don't get 72.75% mIOU in val. My result is about 69.5%-70% mIOU in val. Can you give me your pretrained or your training code is right?

**Comment by lorenmt** [comment]
Have you changed any hyper-parameters? 

My three random seeds gave 73.08, 73.84, 70.57 respectively. So 70% mIoU is not that out of reach.

**Comment by lorenmt** [comment]
I am closing this issue since no further updates are available.
